buddyboss/buddyboss-platform
# Fixed - Group name is not syncing between BuddyBoss and LearnDash

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1327 .

### How to test the changes in this Pull Request:

1. Make sure that you activated Enable group sync functionality FROM **BuddyBoss Social Groups TO LearnDash Groups** and Enable group sync functionality FROM **LearnDash Groups TO BuddyBoss Social Groups**.
(BuddyBoss > Integration > LearnDash)
2. Create a Group in LearnDash LMS > Groups. Example name: "Group Sync"
3. Go to the frontend, go to Groups page. Then, select the Group Sync, go to Manage, and try to edit the name.
4. Go back at the backend, go to LearnDash > Groups. You will see that it still has the old name which in my testing "Group Sync" instead of the new name "Sync this Groups chochocho".

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crVTqk9soP

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
